templates/base.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block title %}
+      <title>Taxi Service</title>
+    {% endblock title %}
+</head>
+<body>
+  {% block sidebar %}
```

Comment:
Try to add 
```
<body>
  <div class="container-fluid">
    <div class="row">
      <div class="col-sm-3">
``` 
to make it work as a sidebar wich have to be one the side

Reply:
Please read the task, we should't do this

Reply:
You can try that